### PR TITLE
SQL Thing

### PR DESCRIPTION
**Pull request by MTS11648**
has a bug where it can enter infinite loop, will fix later

### CODE REVIEW

File: AutoBotCSharp/Agent.cs [hunk 1]
```diff
@@ -3233,12 +3236,28 @@ public static bool UpdateDBase(string command)
...
+                            Console.WriteLine("SENT PREVIOUSLY ORPHANED MSQL STRING " + command);
+                            File.AppendAllText(logPath, "================ COMMANDS SENT AT " + timestamp + " ================"); // Updates a log file for when the lost leads should be sent.
+                        }
+                        File.Delete(path); // Clean up when we're done.
+                    }
                 }
                 catch (Exception ex)
                 {
-                    Console.WriteLine(ex);
-                    myConnection.Close();
```
Comment (SpadesNeil) [inline AutoBotCSharp/Agent.cs]: Why is this removed? We still want to log the console and close the failed SQL connection.

Comment (MTS11648) [inline AutoBotCSharp/Agent.cs]: I don't know, if I removed that it was an accident.

Comment (SpadesNeil) [inline AutoBotCSharp/Agent.cs]: Looks like an accident to me.

File: AutoBotCSharp/Agent.cs [hunk 2]
```diff
@@ -3233,12 +3236,28 @@ public static bool UpdateDBase(string command)
...
+                    if (Agent.AutoFormQuestion == Agent.TCPA || Agent.AutoFormQuestion == Agent.TCPA2)
```
Comment (SpadesNeil) [inline AutoBotCSharp/Agent.cs]: Is there any reason we put this check here? UpdateDBase() is called, I am pretty sure we want the data to be sent (and logged), ergo we also want it to be saved if we _can't_ send it.

Comment (MTS11648) [inline AutoBotCSharp/Agent.cs]: That's true, it really doesn't _need_ to be there, just habit to put in checks like that.

Comment (SpadesNeil) [inline AutoBotCSharp/Agent.cs]: I'll probably remove this tomorrow. It seems like we shouldn't have this check. That could be what is buggering things.

**Comment by MTS11648** [comment]
Closing because #132 will take its place.
